infra/bff/friends/transcribe.bff.ts

```
@@ -0,0 +1,95 @@
+import { register } from "infra/bff/mod.ts";
+import startSpinner from "lib/terminalSpinner.ts";
+import { transcribeAudio } from "packages/transcribing/transcribeAudio.ts";
+import { getLogger } from "deps.ts";
+import { parseArgs } from "infra/bff/deps.ts";
+import { IBfCurrentViewerInternalAdminOmni } from "packages/bfDb/classes/BfCurrentViewer.ts";
+import { AssemblyAI } from "packages/transcribing/deps.ts";
+import { BfMediaTranscript } from "packages/bfDb/models/BfMediaTranscript.ts";
+
+import { BfMedia } from "packages/bfDb/models/BfMedia.ts";
+import { BfAnyid } from "packages/bfDb/classes/BfBaseModelIdTypes.ts";
+import { BfEdge } from "packages/bfDb/coreModels/BfEdge.ts";
+
+const logger = getLogger(import.meta);
+
+const ASSEMBLY_AI_KEY = Deno.env.get("ASSEMBLY_AI_KEY");
```

Probably better to get this at runtime (ie when you run transcribe) so you can throw if it's not set instead of having to crash the whole app?